<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemonstrationWindow</class>
 <widget class="QWidget" name="DemonstrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Particles in Box</string>
  </property>
  <widget class="QSlider" name="sliderState">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>565</y>
     <width>870</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlay">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>600</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>▯▯</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="comboFps">
   <property name="geometry">
    <rect>
     <x>822</x>
     <y>600</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelFps">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>600</y>
     <width>143</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>600</y>
     <width>220</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Time elapsed: {time:.5f}s</string>
   </property>
  </widget>
  <widget class="ParticleWidget" name="canvas">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>870</width>
     <height>550</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plotMaxwell" native="true">
   <property name="geometry">
    <rect>
     <x>875</x>
     <y>5</y>
     <width>320</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plotBoltzmann" native="true">
   <property name="geometry">
    <rect>
     <x>875</x>
     <y>280</y>
     <width>320</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxwellNumBins">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>260</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="boltzmannNumBins">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>530</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBinNumMaxwell">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>71</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin number</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBinNumBoltzmann">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>510</y>
     <width>71</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExportMaxwell">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>260</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to PNG</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxwellPictureWidth">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaxwellPictureHeight">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxwellWidth">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>260</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxwellHeight">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>260</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoltzmannPictureWidth">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>510</y>
     <width>33</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoltzmannPictureHeight">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>510</y>
     <width>33</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="boltzmannWidth">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>530</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="boltzmannHeight">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>530</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExportBoltzmann">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>530</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to PNG</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParticleWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>src/particlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>src/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
